LeRobotDataset v3.0 conversion (#522)
enactic/openarm: website/docs/tutorial/data-collection.mdx

│   │   |  └── ...
```

> Also, you can convert to the LeRobot dataset v3.0 format by changing the `--format` argument to `lerobot_v3.0`. The v3.0 format is the latest version of the LeRobot dataset format, which has some changes in the file structure and metadata format. Please refer to the [LeRobot dataset documentation](https://huggingface.co/docs/lerobot/lerobot-dataset-v3) for more details on the dataset format.

Reviewer: ```suggestion
Also, you can convert to the LeRobotDataset v3.0 format by changing the `--format` argument to `lerobot_v3.0`. The v3.0 format is the latest version of the LeRobotDataset format, which has some changes in the file structure and metadata format. Please refer to the [LeRobotDataset v3.0 documentation](https://huggingface.co/docs/lerobot/lerobot-dataset-v3) for more details on the dataset format.
```

> Also, you can convert to the LeRobot dataset v3.0 format by changing the `--format` argument to `lerobot_v3.0`. The v3.0 format is the latest version of the LeRobot dataset format, which has some changes in the file structure and metadata format. Please refer to the [LeRobot dataset documentation](https://lerobot.readthedocs.io/en/latest/dataset_format.html) for more details on the dataset format.

Reviewer 2: ```suggestion
Also, you can convert to the LeRobot dataset v3.0 format by changing the `--format` argument to `lerobot_v3.0`. The v3.0 format is the latest version of the LeRobot dataset format, which has some changes in the file structure and metadata format. Please refer to the [LeRobot dataset documentation](https://huggingface.co/docs/lerobot/lerobot-dataset-v3) for more details on the dataset format.
```

Since this is the official one, maybe better to use this?

> ```

Reviewer 2: ```suggestion
```bash
```

uv run openarm-dataset-convert path/to/collected_dataset_path path/to/output_path --format lerobot_v3.0
```

> Please check LeRobot document for v3.0 format. https://huggingface.co/docs/lerobot/lerobot-dataset-v3 

Reviewer: We may want to remove this sentence because the previous paragraph has a similar sentence. 

```suggestion
Please check LeRobotDocument for v3.0 format. https://huggingface.co/docs/lerobot/lerobot-dataset-v3 
```
